lib/gems/pending/openstack/openstack_event_monitor.rb

```diff
@@ -1,129 +0,0 @@
-# The OpentstackEventMonitor uses a plugin pattern to instantiate the correct
-# subclass as a plugin based on the #available? class method implemented in each
-# subclass
-require 'more_core_extensions/core_ext/hash'
-require 'util/extensions/miq-module'
-require 'active_support/core_ext/class/subclasses'
```

Comment:
`activesupport` should be listed as a runtime dependency on `manageiq-providers-openstack`